homeassistant/components/switchbee/button.py

```diff
@@ -0,0 +1,50 @@
+"""Support for SwitchBee scenario button."""
+
+from switchbee.api import SwitchBeeError
+from switchbee.device import ApiStateCommand, DeviceType, SwitchBeeBaseDevice
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import SwitchBeeCoordinator
+from .const import DOMAIN
```

Comment:
The coordinator was moved to a separate module
```suggestion
from .const import DOMAIN
from .coordinator import SwitchBeeCoordinator
```